### PR TITLE
FOIA(dot)gov-pointer

### PR DESCRIPTION
**Pull request by rjmajma**
DO NOT MERGE!

Updated this to point users to the learn section of FOIA.gov in case we didn't answer their question on our page.

### CODE REVIEW

**Comment by rjmajma** [comment]
Gabriel promised me I wouldn't break anything if I did this. 

It's all broken.

File: foia_hub/templates/learn.html [hunk 1]
```diff
@@ -25,6 +25,8 @@
         </ul>
       </nav>
 
+        <h2>Didn't find the information you're looking for? Check <a href="FOIA.gov/about.html">FOIA.gov/Learn</a></h2>
```
Comment (khandelwal) [inline foia_hub/templates/learn.html]: OK, so this is completely crazy but that URL should be all lowercase. It's foia.gov, not FOIA.gov when dealing with the URL. URLs are case sensitive: http://www.w3.org/TR/WD-html40-970708/htmlweb.html

**Comment by ramirezg** [comment]
I won't look nice, help.
css - imgur

**Comment by rjmajma** [comment]
Oh, thanks Travis. Glad to know we can merge this. You're a lot of help.

**Comment by ramirezg** [comment]
It's either way, it's not going into master so nothing would happen.

File: foia_hub/templates/learn.html (hunk 1 above)
Comment (theresaanna) [inline foia_hub/templates/learn.html]: `<a href="http://foia.gov/about.html">` or replace http with https if that's your thing.

**Comment by theresaanna** [comment]
This is the worst PR I've ever seen. What are you guys doing over there!? :clap:

**Comment by rjmajma** [comment]
Ouch. My soul.

**Comment by theresaanna** [comment]
A+ high five magic stars all around

:star: :star2: :dizzy: :rainbow: 

See, I'm nice.

File: foia_hub/templates/learn.html (hunk 1 above)
Comment (theresaanna) [inline foia_hub/templates/learn.html]: Nitpick: your indentation is wonky. Doesn't align with `</nav>`.

Comment (konklone) [inline foia_hub/templates/learn.html]: > URLs are case sensitive: http://www.w3.org/TR/WD-html40-970708/htmlweb.html

URL paths are case sensitive, but domains are case insensitive: https://tools.ietf.org/html/rfc4343

**Pull request closed without merging**